lib/Fhp/Dialog/Dialog.php

```diff
@@ -437,43 +435,31 @@ public function syncDialog($sendHKTan = true, $tanMechanism = null)
 			$this->productVersion
 		);
 
-		if ($sendHKTan) {
-			$options = array();
-			if ($tanMechanism) {
-				$options[AbstractMessage::OPT_PINTAN_MECH] = array($tanMechanism);
-			}
+		$options = array();
+		$encryptedSegments = array(
```

Comment:
This is just refactoring of two large if/else blocks

Reply:
:heavy_check_mark: